.rubocop.yml

```diff
@@ -6,6 +6,9 @@ AllCops:
   NewCops: enable
   TargetRubyVersion: 2.7
 
+Gemspec/DevelopmentDependencies:
+  EnforcedStyle: gemspec
```

Comment:
This is fine, but I don't see a reason why we can't move the dev dependencies into the Gemfile though. I'm all for consistency 😄